.gitignore

```diff
@@ -169,6 +169,7 @@ cython_debug/
 .idea/**/usage.statistics.xml
 .idea/**/dictionaries
 .idea/**/shelf
+.idea/misc.xml
```

Comment:
Don't know why, but this file keeps in my changes although its in the .gitignore.